packages/block/src/header.ts

```diff
@@ -225,6 +226,15 @@ export class BlockHeader {
       }
     }
 
+    // Set these values as constants after the Merge
+    if (this._common.isActivatedEIP(3675)) {
+      uncleHash = KECCAK256_RLP_ARRAY
+      difficulty = new BN(0)
+      extraData = Buffer.from('80', 'hex')
```

Comment:
i know the spec is a little confusing here but for our library we would just set this extraData to an empty buffer: `Buffer.from([])`

Reply:
I think we generally don't want to set this here so harsh that we always overwrite the initially provided values but rather either expect uses to provide the correct values if they want to provide themselves (and throw if incorrect) or otherwise set as the default value if not provided. Otherwise we let through bogus blocks if e.g. passed in as serialized data.

Will update along with Ryan's comment.